### DISCUSSION TITLE
stuck on LineageOS boot screen issue, fixed

### DISCUSSION THREAD

**Discussion by durango99** [pgale/lineage_15.1_Installation_TSV#6] [General]
This is for anyone that has run into flashing the LineageOS image after installing the A11 one.

I had successfully flashed the a11 and decided to go with LineageOS.
I went through the steps when installing LineageOS and it would just get stuck at the bootscreen.  I also went to the stock Teams and Others image and also it would just get stuck at the Lenovo bootscreen.

I saw the following thread about differing partition sizes and decided to use the image suggested in that comment.  I made sure that the stock image got past the bootscreen.  Then I retried flashing LineageOS and within a few minutes it got past the bootscreen.
 
https://community.home-assistant.io/t/is-this-the-perfect-standalone-tablet-for-ha/658422/1092
https://s3.wasabisys.com/filestash-buk/lenovo-thinksmart-view/CD-18781Y.200628.084.zip
 
So not sure which step works:
- using this 200628.084 image?
- using QFIL with admin rights?

**Comment by pgale** [comment]
I think running the RSA tool to recover might also work. Guess it restores partition sizes maybe.
